profiles/factory.py

```diff
@@ -24,11 +25,14 @@ def create_app(config: Config, clients: Clients) -> Flask:
     orcid_client = OrcidClient(config.orcid['api_uri'])
     orcid_tokens = SQLAlchemyOrcidTokens(db)
     profiles = SQLAlchemyProfiles(db)
+
+    publisher = get_publisher(pub_name='profiles', config={**config.bus})
```

Comment:
`**config.bus` is a bit magic, could be explicit and remove `env` (since it's duplicating `profiles.environment`).

Reply:
It was left to unpack due to the fact you can pass in additional arguments such as overriding `endpoint_url` and what not for the boto resource. (previous incarnations of this functionality included this so I did not want to get rid of it) 
```
# SNSPublisher
def __init__(self, region: str = '', subscriber: str = '', name: str = '', env: str = '', **overrides):
```
Though in `Profiles` at least its probably not going to need anything other than the values to build the Arn, so I don't mind being explicit. Will change. Will also using existing `profiles.env` value.